docs/gpa.md

```diff
@@ -157,7 +158,14 @@ This requires the [Additional Contact Points](https://github.com/open-contractin
         <td>where, pursuant to Article IX, a procuring entity intends to select a limited number of qualified suppliers to be invited to tender, the criteria that will be used to select them and, where applicable, any limitation on the number of suppliers that will be permitted to tender; and</td>
         <td markdown=1>
 
-* Add this to `tender/eligibilityCriteria`
+* Set `tender.techniques.hasFrameworkAgreement` to `true`
+* Enter *the number of suppliers that will be permitted to tender* in `tender.techniques.frameworkAgreement.maximumParticipants`
+
+This requires the [Techniques](https://github.com/open-contracting-extensions/ocds_techniques_extension) extension.
+
+* Add *the criteria that will be used to select [the suppliers]* to `tender.selectionCriteria.criteria`
```

Comment:
Let's give both options for structured and unstructured data. Unstructured data can simply use `tender.selectionCriteria.description`.

Reply:
Done